client/src/pages/SignUpPage/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { useSetRecoilState } from 'recoil';
+import { toast } from 'react-toastify';
+import whiteAdd from '@public/icons/white-add.svg';
+import { Button } from '@/lib/design';
+import * as S from './style';
+import user from '@/recoil/user';
+import { errorMessage } from '@/lib/common/message';
+import { signUp } from '@/lib/api/user';
+import { UserState } from '@/contexts/userContext';
+import { taskSortByUpdate } from '@/lib/utils/sort';
+
+export const SignUpPage = () => {
+  const [name, setName] = useState('');
+  const [job, setJob] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [checkPassword, setCheckPassword] = useState('');
+  const [organization, setOrganization] = useState('');
```

Comment:
```suggestion
  const [newUser, setNewUser] = useState<newUser>({})
```
객체로 묶어서 관리하는 것은 어떨까요??